base/doc/usrguide.tex

```diff
@@ -794,6 +794,59 @@ \subsection{Commands at the start of tabular cells}
 \end{tabular}
 \end{verbatim}
 
+\subsection{Using the verbatim argument types}
```

Comment:
Overall comment on this subsection and the next.

It is a bit confusing so that working out where useful  information appears, and exactly what it applies to (v, and/or v+).